core/src/actions/properties.ts

```diff
@@ -333,7 +333,7 @@ export class PropertyProfilePreview extends AuthenticatedAction {
     }
 
     const apiData = await profile.apiData();
-    const source = await property.$get("source");
+    const source = await property.$get("source", { scope: null });
```

Comment:
Not sure about this one - if the source isn't `ready`, should we not be able to preview any properties against it?

Reply:
If you went to the page of a property that had its source `deleted`, the preview would have an error on it saying that you can't call `importProfileProperty` on `null`. However, the error doesn't break anything else and it's just displayed for that property in the preview. Do you think we should instead just have a better error message for when the source isn't ready?